stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/common/Extensions.kt

```diff
@@ -110,7 +110,7 @@ val ChannelData.name: String
     get() = (extraData["name"] as? String) ?: ""
 
 val Member.isOwner: Boolean
```

Comment:
I don't know if this still works properly or not or if it serves the same use case, we should probably merge to capabilities soon